src/main/java/org/jenkinsci/plugins/saml/UpdateMetadataFromURLPeriodicWork.java

```diff
@@ -58,8 +58,8 @@ public long getInitialDelay() {
      */
     private long getConfiguredPeriod() {
         long ret = 0;
-        jenkins.model.Jenkins j = jenkins.model.Jenkins.getInstance();
-        if (j.getSecurityRealm() instanceof SamlSecurityRealm) {
+        jenkins.model.Jenkins j = jenkins.model.Jenkins.getInstanceOrNull();
```

Comment:
Why you change from `jenkins.model.Jenkins.getInstance()` to  `jenkins.model.Jenkins.getInstanceOrNull()`?

Reply:
I guess this change is motivated because Jenkins.getInstance() is deprecated, the logic change it is to Jenkins.get() no to Jenkins.getInstanceOrNull()

Reply:
Thank you didn't saw this - I only saw that the getInstance() was calling getInstanceOrNull in the internals, will change this.